test/client-unit/components/bhModalNotify.spec.js

```diff
@@ -0,0 +1,181 @@
+    const compiled = $compile(angular.element(componentTemplate))($scope);
+    $scope.someValue = 'Info Message';
+    const translated = $translate.instant($scope.someValue);
+    $scope.$digest();
+
+    const notificationClass = find(compiled, '.notification-info');
+    const message = find(angular.element(notificationClass), 'span');
+    const text = angular.element(message).text();
+    expect(notificationClass).to.exist;
+    expect(translated).to.equal(text);
+  });
+
+  it('shows success message with success style when the type is given', () => {
+    const componentTemplate = `
+      <bh-modal-notify type="success" value="someValue"></bh-modal-nofify>
+    `;
+    const compiled = $compile(angular.element(componentTemplate))($scope);
+    $scope.someValue = 'Success Message';
+    const translated = $translate.instant($scope.someValue);
+    $scope.$digest();
+
+    const notificationClass = find(compiled, '.notification-success');
+    const message = find(angular.element(notificationClass), 'span');
+    const text = angular.element(message).text();
+    expect(notificationClass).to.exist;
+    expect(translated).to.equal(text);
+  });
+
+  it('shows warning message with warn style when the type is given', () => {
+    const componentTemplate = `
+      <bh-modal-notify type="warn" value="someValue"></bh-modal-nofify>
+    `;
+    const compiled = $compile(angular.element(componentTemplate))($scope);
+    $scope.someValue = 'Warning Message';
+    const translated = $translate.instant($scope.someValue);
+    $scope.$digest();
+
+    const notificationClass = find(compiled, '.notification-warn');
+    const message = find(angular.element(notificationClass), 'span');
+    const text = angular.element(message).text();
+    expect(notificationClass).to.exist;
+    expect(translated).to.equal(text);
+  });
+
+  it('shows error message with error style when the type is given', () => {
+    const componentTemplate = `
+      <bh-modal-notify type="error" value="someValue"></bh-modal-nofify>
+    `;
+    const compiled = $compile(angular.element(componentTemplate))($scope);
+    $scope.someValue = 'Error Message';
+    const translated = $translate.instant($scope.someValue);
+    $scope.$digest();
+
+    const notificationClass = find(compiled, '.notification-error');
+    const message = find(angular.element(notificationClass), 'span');
+    const text = angular.element(message).text();
+    expect(notificationClass).to.exist;
+    expect(translated).to.equal(text);
+  });
+
+  it('shows danger message with danger style when the type is given', () => {
+    const componentTemplate = `
+      <bh-modal-notify type="danger" value="someValue"></bh-modal-nofify>
+    `;
+    const compiled = $compile(angular.element(componentTemplate))($scope);
+    $scope.someValue = 'Danger Message';
+    const translated = $translate.instant($scope.someValue);
+    $scope.$digest();
+
+    const notificationClass = find(compiled, '.notification-danger');
```

Comment:
These tests are great!  I think they are fine as they are.  One potential problem is that we may never find the `.notification-danger` class if a bug occurs where danger becomes success.  A way to test for this would be to instead search for the `.notification` class, then expect it to also have `.notification-danger`, like we do for the `fa-times` class.

```js
const notification  = find(compiled, '.notification');
expect(angular.element(notification).hasClass('notification-danger')).to.equal(true);
```